integration/dbt/scripts/dbt-ol

```diff
@@ -53,9 +54,10 @@ def main():
 
     events = processor.parse().events()
 
-    for event in events:
+    for event in tqdm(events, desc="Emitting OpenLineage events:"):
```

Comment:
it looks like ":" is not needed here. Looking at your recording there are two in the display